src/gwproto/types/component_attribute_class_gt.py

```diff
@@ -14,3 +14,5 @@ class ComponentAttributeClassGt(BaseModel):
     MakeModel: MakeModel
     TypeName: Literal["component.attribute.class.gt"] = "component.attribute.class.gt"
     Version: Literal["000"] = "000"
+
+    model_config = ConfigDict(extra="allow")
```

Comment:
Upon reflection, I propose we don't do this; we leave default of extra="ignore" and instead we add: 
```
class AnyComponentAttributeClassGt(ComponentAttributeClassGt):
    """Class for loading unrecorgnized cac types"""
    TypeName: str
    
    model_config = ConfigDict(extra="allow")

```
and then using that in the explicit existing case where we try to decode a cac/component and we fail because we don't recognize the type

Reply:
I thought I had added it and had a test for it, but either I didn't or it got lost in my merge.

Reply:
I prefer to keep extra="allow" for all of the Cacs.